src/meta/client/MetaClient.cpp

```diff
@@ -183,13 +183,13 @@ bool MetaClient::loadData() {
     }
     diff(oldCache, localCache_);
     ready_ = true;
-    LOG(INFO) << "Load data completed!";
```

Comment:
I think it's useful here, use LOG_EVERY_N

Reply:
I print in line 192, INFO

Reply:
192L just means he's added to the task queue, not that he's actually called